CHANGELOG.md

```diff
@@ -4242,3 +4243,4 @@
 [@jfhinchcliffe]: https://github.com/jfhinchcliffe
 [@jdkaplan]: https://github.com/jdkaplan
 [@cstyles]: https://github.com/cstyles
+[@avmnu-sng]: https://github.com/username
```

Comment:
It should be replaced with your account name.

Reply:
How silly! Fixed